packages/console/src/components/RolePermissionsTransfer/components/ResourceItem/index.tsx

```diff
@@ -70,7 +70,9 @@ const ResourceItem = ({
             key={scope.id}
             scope={scope}
             isSelected={selectedPermissions.findIndex(({ id }) => scope.id === id) >= 0}
-            onSelectPermission={onSelectPermission}
+            onSelect={() => {
+              onSelectPermission(scope);
```

Comment:
nit: Prefer the original approach. Since you have to pass in the scope anyway, the scope from the prop could be the only reference.

Reply:
updated.